### PR TITLE
Update types to Cosmos SDK 0.43 RC2

### PR DESCRIPTION
**Pull request by webmaster128**

### CODE REVIEW

File: src/cosmos/upgrade/v1beta1/query.ts [hunk 1]
```diff
@@ -56,7 +55,29 @@ export interface QueryUpgradedConsensusStateRequest {
  * RPC method.
  */
 export interface QueryUpgradedConsensusStateResponse {
-  upgradedConsensusState?: Any;
+  upgradedConsensusState: Uint8Array;
```
Comment (webmaster128) [inline src/cosmos/upgrade/v1beta1/query.ts:58]: This seems to be a breaking change

Comment (robert-zaremba) [reply src/cosmos/upgrade/v1beta1/query.ts:58]: Good catch. It seams however that this is not used - nobody is calling `SetUpgradedConsensusState`, so there is no response.

Comment (robert-zaremba) [reply src/cosmos/upgrade/v1beta1/query.ts:58]: It was changed here: https://github.com/cosmos/cosmos-sdk/pull/8673/files#r682168269

**Comment by webmaster128** [comment]
Outdated. Types are generated from 0.44 now.

**Pull request closed without merging**